handlers/__init__.py

```diff
@@ -249,6 +249,7 @@ def get_cookie_secret():
     scoreboard_state={},
     # Application version
     version=__version__,
+    autoreload=options.autoreload_source
```

Comment:
Not necessarily... you only need to disable auto reload if you’re connecting a debugger. If you’re just modifying single files and using trace logs instead of something with breakpoints, having the system auto reload is helpful, and probably how devs have done things so far.

Reply:
Okay, my misunderstanding.